pkgs/by-name/u3/u3-tool/package.nix

```diff
@@ -1,16 +1,17 @@
 { lib, stdenv, fetchurl }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "u3-tool";
-  version = "0.3";
+  version = "1.0";
 
   enableParallelBuilding = true;
 
   src = fetchurl {
-    url = "mirror://sourceforge/${pname}/${pname}-${version}.tar.gz";
+    url = "mirror://sourceforge/${finalAttrs.pname}/${finalAttrs.pname}-${finalAttrs.version}.tar.gz";
     sha256 = "1p9c9kibd1pdbdfa0nd0i3n7bvzi3xg0chm38jg3xfl8gsn0390f";
```

Comment:
All mirrors return 404 for this URL

Reply:
you are right, i fogor to update checksum which resulted in a cache hit :)